doc/whats_new/v0.24.rst

```diff
@@ -195,6 +195,13 @@ Changelog
   usage in `fit`. :pr:`18334` by `Olivier Grisel`_ `Nicolas Hug`_, `Thomas
   Fan`_ and `Andreas Müller`_.
 
+- |Efficiency| Histogram initialization is now done in parallel in
+  :class:`ensemble.HistGradientBoostingRegressor` and
+  :class:`ensemble.HistGradientBoostingClassifier` which results in speed
+  improvement for problems that build a lot of nodes on multicore machines.
+  :pr:`18341` by `Olivier Grisel`_, `Nicolas Hug`_, `Thomas Fan`_, and
+  :user:`Egor Smirnov <SmirnovEgorRu>`.
```

Comment:
Crediting @SmirnovEgorRu here since you opened a similar PR a while ago https://github.com/scikit-learn/scikit-learn/pull/14380